src/get-own-property-symbols.js

```diff
@@ -89,7 +90,7 @@
         }
       };
       defineProperty(ObjectProto, uid, descriptor);
-      return (source[uid] = defineProperty(
+      return freeze(source[uid] = defineProperty(
```

Comment:
if you drop freeze you should drop the internal reference. I've used to be sure that object cannot possibly be changed by any other library